src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2016 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Tizen.Network.Stc
+{
+    /// <summary>
+    /// A class for managing the Stc Info (statistics information).
+    /// </summary>
+    /// <since_tizen> tizen_5.5 </since_tizen>
+
+    public class StcInfo : IDisposable
+    {
+        private IntPtr _infoHandle = IntPtr.Zero;
+        private bool _disposed = false;
+
+        internal StcInfo(IntPtr handle)
+        {
+            Log.Debug(Globals.LogTag, "New Info. Handle: " + handle);
+            _infoHandle = handle;
+        }
+
+        ~StcInfo()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// A method to destroy the managed StcInfo objects.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
```

Comment:
Current Platform version is 5.5 and API version is 6. Please change it to <since_tizen> 6 </since_tizen>

Reply:
Done as per your suggestion.